test/e2e/rte/rte_e2e_suite_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package rte_e2e
+package rte
```

Comment:
why?

Reply:
sure, but let's have either `_e2e` in the package name or move it back under `test/e2e` like you already want to do (https://github.com/openshift-kni/numaresources-operator/pull/48#discussion_r765575929) . I prefer the latter.

Reply:
you can see that RTE tests are already placed under `test/e2e/rte` does it good enough or do you want some additional changes?

Reply:
nope, that's good enough thanks!